service/lib/dinstaller/dbus/manager.rb

```diff
@@ -60,6 +60,7 @@ def initialize(backend, logger)
         dbus_method(:Probe, "") { config_phase }
         dbus_method(:Commit, "") { install_phase }
         dbus_method(:CanInstall, "out result:b") { can_install? }
+        dbus_method(:ProvideLogs, "out tarball_filesystem_path:s, in user:s") { |u| provide_logs(u) }
```

Comment:
To make it extensible in the future I'd rather return a hash or array of strings here.

Reply:
But let's start with a simple API and extend it if needed later.

Reply:
As said on IRC I find it hard to use when it is array or hash as usual action for logs will be display or download->upload to bzilla or gh issues, and in such case having more than one file is badly usable. What can be used in future is hash for another options if we want e.g. filter out debug output or logs just for specific parts like web logs, but I think we can extend it in future if the need grows. So far simple file that download/upload user should be enough